docs/wallet_API.md

```diff
@@ -271,12 +271,12 @@ The class constructor takes one parameter, the name of your dapp (this will be u
 
 ```
 
-await wallet.connect('mainnet' | 'delphinet' | 'carthagenet' | 'labnet' | 'sandbox');
+await wallet.connect('mainnet' | 'edonet' | 'delphinet' | 'labnet' | 'sandbox');
```

Comment:
I think Temple write edo2net instead of edonet: https://github.com/madfish-solutions/templewallet-dapp/blob/master/src/types.ts#L115

Reply:
good catch! fixed